index.html

```diff
@@ -31,7 +31,167 @@ <h1>Charlie Giorgania</h1>
+
+        <div class="container3">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg4"></div>
+
+        <div class="container4">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg5"></div>
+
+        <div class="container5">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+
+        <div class="projImg6"></div>
+
+        <div class="container6">
+          <p class="projName">Project name goes here</p>
+          <ul class="stacks">
+            <li>HTML/CSS</li>
+            <li>Ruby on Rails</li>
+            <li>JavaScript</li>
+          </ul>
+
+          <button class="projView">
+            See this Project
+            <img class="arrow" src="./images/arrow.png" alt="" />
+          </button>
+        </div>
+      </section>
+
+      <section>
+        <div class="aboutMe"><h2>About</h2></div>
+        <p class="aboutDescription">
+          Lorem ipsum dolor sit amet, consectetur <br />
+          adipiscing elit, sed do eiusmod tempor<br />
+          incididunt ut labore et dolore magna <br />
+          aliqua. Ut enim ad minim veniam,quis<br />
+          nostrud exercitation ullamco laboris nisi<br />
+          ut aliquip ea eiusmod
+        </p>
+
+        <div class="aboutButton">
+          <p>Get my CV</p>
+          <img class="aboutArrow1" src="./images/aboutarrow1.png" alt="" />
+          <img class="aboutArrow" src="./images/aboutarrow.png" alt="" />
+        </div>
+
+        <div class="languages">
+          <p>Languages</p>
+          <ul>
+            <li>JavaScript (ES5/ES6)</li>
+            <li>Ruby</li>
+            <li>HTML</li>
+            <li>CSS</li>
+          </ul>
+          <img class="languageTag" src="./images/abouttag.png" alt="" />
+        </div>
+
+        <div class="frameworks">
+          <p>Frameworks</p>
+          <ul>
+            <li>Bootstrap</li>
+            <li>Ruby on rails</li>
+            <li>RSpec</li>
+            <li>Capibara</li>
```

Comment:
- Please the font-size of the names of technology is supposed to be 18px but your is 15px please use the appropriate font-size as it is on the figma design. 

##### Your design 
![Screenshot from 2022-07-02 17-17-34](https://user-images.githubusercontent.com/58771507/177008402-23a55948-d543-4c92-8e7c-2441fbdc6046.png)

##### Figma design

![Screenshot from 2022-07-02 17-16-53](https://user-images.githubusercontent.com/58771507/177008423-ba5dc230-ad31-4d43-9a53-f5e42880b23f.png)

Reply:
I have fixed the font size for technologies